### DISCUSSION TITLE
Installing and uninstalling seemingly not possible

### DISCUSSION THREAD

**Discussion by klettervirus** [toltec-dev/toltec#654] [Support] [answered]
I tried to install toltec, but obviously did that before some years ago. On my rm1. Software Version 2.7.0.51
The following happens, when I 1) try to install it and 2) try to remove it:

```
╺━┓┏━╸┏━┓┏━┓   ┏━╸┏━┓┏━┓╻ ╻╻╺┳╸┏━┓┏━┓
┏━┛┣╸ ┣┳┛┃ ┃   ┃╺┓┣┳┛┣━┫┃┏┛┃ ┃ ┣━┫┗━┓
┗━╸┗━╸╹┗╸┗━┛   ┗━┛╹┗╸╹ ╹┗┛ ╹ ╹ ╹ ╹┗━┛
reMarkable: ~/ toltecctl uninstall
-sh: toltecctl: command not found
reMarkable: ~/ wget http://toltec-dev.org/bootstrap
Connecting to toltec-dev.org (5.161.99.157:80)
wget: can't open 'bootstrap': File exists

```
Can someone please point me to a way to remove any residue from previous installs so I can get a new clean install? Sorry, my Linux knowledge is only tiny, I can use a command line, but have no idea how to get a handle on this one.
Thank for all the effort and support!

**Comment by matteodelabre** [comment, marked as answer]
Have you tried removing the existing `bootstrap` file first? (`rm bootstrap`)

**Reply by klettervirus** [reply]
Yes, that did the trick, thank you! Unfortunately there seems to be another issue (not sure if this is the place to talk about it though). On installing it cannot remove /opt and the script is not installed:

```
bootstrap: OK
INFO:  Fetching secure wget
INFO:  Bootstrapping Opkg
2022-11-22 12:12:08 URL:https://bin.entware.net/armv7sf-k3.2/installer/opkg [752476/752476] -> "/home/root/.local/bin/opkg" [1]
2022-11-22 12:12:09 URL:https://toltec-dev.org/stable/rmall/toltec-bootstrap_0.2.2-1_rmall.ipk [6667/6667] -> "/tmp/toltec-bootstrap_0.2.2-1_rmall.ipk" [1]
INFO:  Installing Toltec and Entware
rm: can't remove '/opt': Device or resource busy
ERROR: Unexpected error on line bootstrap:181 in function main
ERROR: This script failed to install. If you can't solve the above
       issue yourself, please report it at:

           <https://github.com/toltec-dev/toltec/issues>

       (Please also include these error logs to help solving the
       problem faster. Thank you!)
Removing mount over '/opt'
Removed /etc/systemd/system/local-fs.target.wants/opt.mount.
reMarkable: ~/ opkg update
-sh: opkg: command not found

```
Should I open another issue with this? I looked in the discussion and in "Issues", but it seems like I am the only one with this issue?! I am also not sure if I should put this Problem into the Issues section. Since it might be connected to the first part above, I will leave it here.

Thanks for all the kind help!

**Reply by Eeems** [reply]
It's a bit late, but did you ever resolve this?
